cronsul

```diff
@@ -39,9 +52,8 @@ esac
 
 command=("$@")
 
-# This will be wrong if cron starts our job really late.
-schedule_date="$(( ($(date '+%s') / period) * period ))"
+current_run="$(date '+%s')"
```

Comment:
Yeah, that's true. I'm using cronsul to synchronize hourly backups and the servers are all using the same ntp host, so I'm using a period of 10 minutes just for safety's sake and feel very comfortable with this.

Periods close to the period of cron should be wrong though -- the period is your maximum allowable drift between servers, not the period that cron is running on, IMO.